memory/main.py

```diff
@@ -2254,6 +2240,12 @@ def save_popup_cursor():
     key = base_value + 0x0146780A
     return process.read_bytes(key, 1)
 
+def save_conf_cursor():
```

Comment:
I think there is a helper to do this stuff. We would just pass in the hex value, nothing else is needed. Can't remember what it's called.

Reply:
I don't know either.